libpysal/examples/remotes.py

```diff
@@ -643,7 +643,7 @@ def _remote_data():
 
     url = "https://geodacenter.github.io/data-and-lab//"
     try:
-        page = requests.get(url)
+        page = requests.get(url, timeout=10)
```

Comment:
```suggestion
        page = requests.get(url, timeout=(10, None))
```

Let's set only the connect timeout and leave the read timeout without a limit. Some examples are quite large and 10s may be too short.